### ISSUE TITLE
Warning since updating to HA 2022.11.0b0

### ISSUE THREAD

**Issue by gieljnssns** [briis/weatherbit#82]
```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:77
First occurred: 09:14:12 (1 occurrences)
Last logged: 09:14:12

Detected integration that accesses the `is_metric` property of the unit system. This is deprecated and will stop working in Home Assistant 2023.1. Please adjust to use instance check instead.. Please report issue to the custom integration author for weatherbit using this method at custom_components/weatherbit/__init__.py, line 72: if hass.config.units.is_metric
```

**Comment by briis** [comment]
Thanks for reporting. Fix is implemented, awaiting 2022.11 release

**Comment by briis** [comment]
1.0.16 released which closes this issue.
